test/unit/services/helpers/test-group-scenario-data.js

```diff
@@ -7,17 +7,18 @@ let scenarioObjectsArray
 describe('services/helpers/group-scenario-data', function () {
   before(function () {
     scenarioObjectsArray = groupScenarioData(scenarioData)
+    console.log(scenarioObjectsArray)
```

Comment:
As above. Do we want this console log.
This log is in tests so not too important.

Reply:
fixed thx :)